
Do not include spinlock.h and all it's freaky stuff for user-space
applications.

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.8.1/include/linux/mc146818rtc.h.orig	2004-09-15 21:41:03.095156462 +0200
+++ linux-2.6.8.1/include/linux/mc146818rtc.h	2004-09-15 21:41:32.440465983 +0200
@@ -13,10 +13,12 @@
 
 #include <asm/io.h>
 #include <linux/rtc.h>			/* get the user-level API */
-#include <linux/spinlock.h>		/* spinlock_t */
 #include <asm/mc146818rtc.h>		/* register access macros */
 
+#ifdef __KERNEL__
+#include <linux/spinlock.h>             /* spinlock_t */
 extern spinlock_t rtc_lock;		/* serialize CMOS RAM access */
+#endif /* __KERNEL__ */
 
 /**********************************************************************
  * register summary
